src/Farmer/Deploy.fs

```diff
@@ -133,7 +156,8 @@ module Az =
         |> String.concat " "
         |> az
     let batchUploadStaticWebsite name path =
-        az $"storage blob upload-batch --account-name %s{name} --destination $web --source %s{path}"
+        let overwriteParameter = checkVersion (Version "2.34.0") |> function | Ok _ -> "--overwrite true" | Error _ -> ""
```

Comment:
Perhaps bind `Version "2.34.0"` into a (private?) symbol in the module? Nothing wrong with leaving it here, but it might be good to put it somewhere with a specific comment about why 2.34.0 is "special".

Reply:
Also consider replacing the use of `function` with a (slightly more verbose but more common) pattern match.

Reply:
Something like this?
```fsharp
    /// The overwrite parameter was introduced in Azure CLI v2.34.0 with a breaking change to the default behaviour
    let private cliVersionWithOverwriteParameter = Version "2.34.0"
    let batchUploadStaticWebsite name path =
        let additionalParameters =
            match checkVersion cliVersionWithOverwriteParameter with
            | Ok _ -> "--overwrite true"
            | _ -> ""
        az $"storage blob upload-batch --account-name %s{name} --destination $web --source %s{path} {additionalParameters}"
```

Reply:
Yep exactly

Reply:
changes pushed